modules/kde/hm.nix

```diff
@@ -220,11 +223,54 @@ let
     printf '%s\n' "$kdeglobals" >"$out/kdeglobals"
   '';
 
+  activator = pkgs.writeShellScriptBin "stylix-set-kde-wallpaper" ''
+    set -eu
+    global_path() {
+      for directory in /run/current-system/sw/bin /usr/bin /bin; do
+        if [[ -f "$directory/$1" ]]; then
+          printf '%s\n' "$directory/$1"
+          return 0
+        fi
+      done
+
+      return 1
+    }
+
+    if wallpaper_image="$(global_path plasma-apply-wallpaperimage)"; then
+      "$wallpaper_image" "${themePackage}/share/wallpapers/stylix"
+    else
+      echo "Skipping plasma-apply-wallpaperimage: command not found"
+    fi
+
+    if look_and_feel="$(global_path plasma-apply-lookandfeel)"; then
+      "$look_and_feel" --apply stylix
+    else
+      echo "Skipping plasma-apply-lookandfeel: command not found"
+    fi
+  '';
+
+  activateDocs = "https://stylix.danth.me/options/hm.html#stylixtargetskdesetter";
```

Comment:
This link goes to `#stylixtargetskdesetter`, but the option declared below is called `stylix.targets.kde.service`. Maybe it should link to `#stylixtargetkdeservice` instead?

Take this with a grain of salt tho, since I have not written any stylix code yet :)

Reply:
Yes, thank you, I renamed the option and forgot to change the link. Will fix when I get home.

Reply:
Done